script2.js

```diff
@@ -0,0 +1,71 @@
+// 1. Programming Basics
+// 'Spaghetti', 'Lasagne', 'Beans on toast', 'Porridge', 'Tomato soup', 'Salmon'
+// Create an array of the dishes listed above. Loop through the array using aforloop.
+// 1.Log out all the dishes.
+// 2.Log out a randomly chosen dish (randomly generated  index)
+
+const dishes = ['Spaghetti', 'Lasagne', 'Beans on toast', 'Porridge', 'Tomato soup', 'Salmon'];
+
+for (let i = 0; i < dishes.length; i++) {
+    console.log(dishes[i]);
+}
+
+const rand = Math.floor(Math.random() * dishes.length);
+console.log(`Random: ${dishes[rand]}`);
+
```

Comment:
Nice! :)
To take it up a level in a simple and easy way, you could put the logic that gets a random integer within a certain range into a separate function and use that 💡